File: Python3/프로그래머스/0/181932. 코드 처리하기/코드 처리하기.py
def solution(code):
    answer = ""
    mode = 0
    for i in range(0, len(code)):
        if code[i] == '1': #code[idx]가 1일 경우
            mode = (mode + 1) % 2
        elif mode == i % 2:
            answer += code[i]
            
    return "EMPTY" if len(answer) == 0 else answer

File: C++/프로그래머스/0/181932. 코드 처리하기/코드 처리하기.cpp
#include <string>
#include <vector>
#include <iostream>

using namespace std;

string solution(string code) {
    int mode = 0;
    string answer = "";
    
    for(int i = 0; i < code.length(); i++) {
        if(code[i] == '1') mode = (mode + 1) % 2;
        else if(mode == i % 2)  answer += code[i];
    }
    return answer.empty() ? "EMPTY" : answer;
}